cmd/loom/loom.go

```diff
@@ -838,6 +838,7 @@ func loadApp(
 		CreateRegistry: createRegistry,
 		Migrations: map[int32]tx_handler.MigrationFunc{
 			1: migrations.DPOSv3Migration,
+			3: migrations.ConsolidateFeaturesMigration,
```

Comment:
Why this is numbered as 3?

Reply:
2nd is reserved for Coin Policy Migration. It will be merged in from Coin Policy PR.